server/src/main.rs

```diff
@@ -64,7 +64,7 @@ struct RestateArguments {
     /// Wipes the configured data before starting Restate.
     ///
     /// **WARNING** all the wiped data will be lost permanently!
-    #[arg(value_enum, long = "wipe")]
+    #[arg(value_enum, long = "wipe", hide = true)]
```

Comment:
Optional but we could also rename this to something a more scary-sounding, like `--destroy-persisted-data`. That will be more work though as we probably want to replace [other occurrences](https://github.com/search?q=org:restatedev%20%22--wipe%22&type=code) where we mention this flag.

Reply:
Hey, I already have another PR open for the `documentation` part. But I can definitely rename it here as well. Sounds good to me

Reply:
Personally, I'd be fine with keeping it as `wipe`. For me, this already sounds scary enough.